recipes-bsp/xt-rpi-u-boot-scr/xt-rpi-u-boot-scr.bb

```diff
@@ -0,0 +1,57 @@
+SUMMARY = "U-boot boot script for Xen on Raspberry Pi 5"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+COMPATIBLE_MACHINE = "^raspberrypi5$"
+
+DEPENDS = "u-boot-mkimage-native"
+
+INHIBIT_DEFAULT_DEPS = "1"
+
+TEMPLATE_FILE = "boot.cmd.xen.in"
+
+SRC_URI = "file://${TEMPLATE_FILE}"
+
+BOOT_MEDIA ?= "mmc"
+DOM0_IMAGE ?= "zephyr.bin"
+DOM0_IMG_ADDR ?= "0xe00000"
+DOMD_IMAGE ?= "Image.gz"
+DOMD_IMG_ADDR ?= "0x1200000"
+DOMD_DTB ?= "${RPI_SOC_FAMILY}-${MACHINE}-domd.dtb"
+DOMD_DTB_ADDR = "0x1c00000"
+XEN_IMAGE ?= "xen"
+XEN_IMG_ADDR ?= "0x4000000"
+XENPOLICY_IMAGE ?= "xenpolicy"
+XENPOLICY_IMG_ADDR ?= "0x4150000"
+XEN_DTB ?= "${RPI_SOC_FAMILY}-${MACHINE}-xen.dtb"
+XEN_DTB_ADDR ?= "0x4200000"
+UBOOT_BOOT_SCRIPT ?= "boot.scr"
+
```

Comment:
Above should have comment section describing how much memory in Mb allocated for each component

Reply:
Added